ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendRestRequestServiceFactory.java

```diff
@@ -90,9 +93,12 @@ public RestRequestService getRestRequestService() {
           new AccountAndContainerInjector(accountService, frontendMetrics, frontendConfig);
       StorageQuotaService storageQuotaService = null;
       if (frontendConfig.enableStorageQuotaService) {
+        // TODO: after enabling StatsReport API, add mysql store to frontend rest request service.
```

Comment:
I do not understand this comment. Where are we enabling StatsReport API in FrontendRestRequestServiceFactory?

Reply:
We are going to expose another HTTP API, /statsReport in frontend, to send back stats report to client. Stats reports are stored in mysql database. AccountStatsStore is the object to interact with mysql database to fetch the stats reports. This API will be handled in FrontendRestRequestService just like any other APIs.That why we need to pass the AccountStatsStore object into the factory. However, StorageQuotaService also need to interact with mysql database, so we have to share the object. That's why I need to create this object outside of storage quota service.